vm/src/import.rs

```diff
@@ -20,8 +21,14 @@ fn import_module(vm: &mut VirtualMachine, module: &String) -> PyResult {
         return Ok(module);
     }
 
+    // Check for Rust-native modules
+    if let Some(module) = stdlib::get_modules(vm).get(module) {
```

Comment:
get_modules is now called often, it's return value should be stored in ctx somewhere or in the vm maybe.